app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="sign-container">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class:"btn-sign" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Please make sure the sign-in page's design adheres to the wireframe. You can completely customize your visual design based on the requirements, but you should maintain the wireframe. 👌 😊

### wireframe :point_down: 

![Screenshot from 2023-09-07 15-07-12](https://github.com/binodbhusal/BlogApp/assets/89228743/d83d8b50-4f19-4acb-a7e9-ffd3913c1862)